lecture/SyntaxCheatSheet.js

```diff
@@ -1,21 +1,57 @@
 // Syntax Cheat sheet
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ //ARRAY MANIPULATION
 var x = 3; // How to declare a variable
-var myList = ['Mahd', 'Blanket' , 'Laptop']; //List (each item seperated by comma)
+let myList = ['Mahd','Blanket','Laptop','CellPhone', 'Wifi']; //List (each item seperated by comma)
+console.log(myList);
+
+let newList = [myList.shift(), myList.pop()]; //Shift gets the first item in the list, pop gets the last
+//console.log(newList); //Logs the first and last items
+//console.log(myList); // Old list is now updated with first and last items out
+newList.push("Sweater") // Adding sweater ot the newList
+//For more information, read on splice https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
+
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 
+//OBJECT MANIPULATION
+let irene = {             //Creating an obect called Irene 
+	'gender': 'Female',
```

Comment:
watch for inconsistent tabs, please modify

Reply:
also, can you use the names of students in the course? I've found it helps with engagement when we refer to students in code